src/languages/ux.coffee

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+
+  name: "UX"
```

Comment:
I'd prefer this be something more specific and description, such as `Fuse UX` or `UX Markup` as shown https://www.fusetools.com/docs/basics/ux-markup-reference